packages/bsky/src/data-plane/server/indexing/plugins/post.ts

```diff
@@ -304,6 +339,7 @@ const deleteFn = async (
       .executeTakeFirst(),
     db.deleteFrom('feed_item').where('postUri', '=', uriStr).executeTakeFirst(),
   ])
+  await db.deleteFrom('quote').where('subject', '=', uriStr).execute()
```

Comment:
I think this is the right place to stick this, deleting all the quotes for the post once we delete it.

Reply:
There's another one of these below.  I think we can nix this one.
```suggestion
```

Reply:
okay looks like w/o this the test fails. let me actually pull down the code and look while i fix up that test.

Reply:
Ohh you know, I think I had it wrong.  The quote gets deleted if either the post or the quoted post get deleted.

Reply:
ah yep. added this line back now.